plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -473,6 +481,8 @@ export class ElasticSearchSearchEngine implements SearchEngine {
       return {
         provider: 'aws',
         node: config.getString('node'),
+        region: config.getOptionalString('region'),
```

Comment:
This one should be added to [the config schema](https://github.com/backstage/backstage/blob/b8ce15b1e5bfa2f5184dc61c96af6c45addcd1aa/plugins/search-backend-module-elasticsearch/config.d.ts#L117). I see it was read below already, but while we're here we could update the schema to be appropriate.

Reply:
yep can do